### PR TITLE
added te clone_lansa_version logic to get the source server name

### PR DESCRIPTION
**Pull request by tarun4931**
Removed the parameter sourceserver, updated the code to get the source server name using the tags clone lansa version tag. added the condition to check if more than one clone lansa version exist and throw an error

### CODE REVIEW

File: Scripts/azure_stack_provision.ps1 [hunk 1]
```diff
@@ -14,19 +12,18 @@ param (
 	[string]$sql_password
 )
 
+#####Server Name############
+$sql_server = "db-regression-$lansa_version" #only accepting lower case
 
-
+#######################Retriving Azure Resources using Tags ##################### 
+$db = "test"
```
Comment (tarun4931) [inline Scripts/azure_stack_provision.ps1:21]: For Azure sql server there is always going to be more than 1 db when you query for the db in that server. Apart from the ones we create, there will be a default db called "master". The reason I am using $(db)

**Review by tarun4931 — COMMENTED** [review]
Tested the changes in pipeline it is able to get the sourceserver
https://dev.azure.com/VisualLansa/Database%20Regression%20Test/_releaseProgress?_a=release-environment-logs&releaseId=78&environmentId=307

**Comment by robe070** [comment]
Is this what you are trying to show me in the 1590 lines of output? This is all there is that is output from your script? (The rest being Azure CLI setup and the like). Far too few log messages. There needs to be more sensible ones output.
```
2022-10-31T14:24:56.0634843Z ResourceGroupName     : dbregressiontest
2022-10-31T14:24:56.0635536Z ServerName            : dbregressiontest
2022-10-31T14:24:56.0637639Z DatabaseName          : test
2022-10-31T14:24:56.0638815Z Location              : eastus
2022-10-31T14:24:56.0641007Z CopyResourceGroupName : dbregressiontest
2022-10-31T14:24:56.0642544Z CopyServerName        : db-regression-150060
2022-10-31T14:24:56.0644301Z CopyDatabaseName      : 150060
2022-10-31T14:24:56.0645915Z CopyLocation          : eastus
2022-10-31T14:24:56.0646648Z CreationDate          : 10/31/2022 2:24:47 PM
```

**Comment by tarun4931** [comment]
Hi Rob,
sure, will update the output to be more descriptive. Before that, can you please conform on the logic part for the $clone_lansa_version to retrieve the $sourceserver If any changes need, will update it 1st.

**Pull request merged**